assets/style.scss.liquid

```diff
@@ -1456,6 +1455,53 @@ body {
   .nav-block li a span { color: #eb2330; padding-left: 2px; }
 }
 
+/*============================================================================
```

Comment:
This all looks great, but I do have one request in an effort to make our lives easier long term. Rather than using the Shogun HTML structure and classes can we use the collection/blog page structure and the classes that apply to that? I think it could cut down on the CSS and make uniformity easier.